scrapers/mmp_network.yml

```diff
@@ -0,0 +1,53 @@
+name: "PovBitch"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fakeshooting.com/video
+      - melonechallenge.com/video
+      - mmpnetwork.com/video
+      - povbitch.com/video
+#      - takevan.com/video
+
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //*[@class="videoTitle"]
+      Details:
+        selector: //div[@class="videoDescription"]
+      Date:
+        selector: //div[@class="videoDate" or @id="videoDate"]/text()
+        postProcess:
+          - replace:
+              - regex: "^Added "
+                with: ""
+          - replace:
+              - regex: " • .*"
+                with: ""
+          - parseDate: January 2, 2006
+      Performers:
+        Name:
+          selector: //div[@class="videoDate"]/a | //div[@class="videoFeaturedModels"]/text()
+          concat: ", "
+          split: ", "
+      Studio:
+        Name:
+          selector: //a[@id="headerNavJoinNow"]/@href
+          postProcess:
+            - replace:
+                - regex: .*signup\.(.+)\.com.*
+                  with: $1
+            - map:
+                fakeshooting: Fake Shooting
+                melonechallenge: Molone Challenge
+                mmpnetwork: MMP Network
+                povbitch: POV Bitch
+                takevan: Take Van
```

Comment:
simpler
```yaml
          selector: //*[@class="logo"]//img/@alt
          postProcess:
            - replace:
                - regex: "'s Logo$"
                  with: ""
```

Reply:
Sadly fakeshootings description for the logo the same as at mmpnetwork: "MMP Network's Logo".